### ISSUE TITLE
There is not state.hdf5 file in the GROMACS/KMC_Methane

### ISSUE THREAD

**Issue by felipeZ** [votca/xtp-tutorials#17]
It is not possible to run the script at [GROMACS/KMC_Methane](https://github.com/votca/xtp-tutorials/tree/master/GROMACS/KMC_Methane) because there is no `state.hdf5` file.

**Comment by JensWehner** [comment]
Yes, I know, this is bad, I do not have the compute power atm to do one. Otherwise we could actually also just copy values from the Methane folder and use a python script to fill values in. Then wo do not have to store the binary file.
